core/controller/src/main/scala/org/apache/openwhisk/core/controller/Actions.scala

```diff
@@ -595,38 +590,6 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
     })
   }
 
-  /**
-   * Constructs a WhiskPackage that is a merger of a package with its packing binding (if any).
-   * This resolves a reference versus an actual package and merge parameters as needed.
-   * Once the package is resolved, the operation is dispatched to the action in the package
-   * namespace.
-   */
-  private def mergeActionWithPackageAndDispatch(method: HttpMethod,
```

Comment:
@upgle Could you elaborate the reason to remove this method a little bit?
It would help people who are not familiar with this change like me to get the intention more easily.

Reply:
If an action is resolved before it's dispatched, there is no way to know its original name in the activate method because it's already resolved.
So, I modified it to resolve inside the activate method after it has been dispatched.

```scala
override def activate(user: Identity, entityName: FullyQualifiedEntityName, env: Option[Parameters])(
    implicit transid: TransactionId) = {	    implicit transid: TransactionId)
```